users/Myo Wai Thnant.md

```diff
@@ -11,9 +11,9 @@ twitter: "https://twitter.com/admanMacker"
 github: "https://github.com/admanMacker"
 linkedin: "https://www.linkedin.com/in/admanMacker"
 company: Byo India 2017
-detail: 'Online Migration: Upgrade with Seamless Downtime'
+detail: ['Online Migration: Upgrade with Seamless Downtime'](https://www.youtube.com/watch?v=HvctSXwu2GI)
```

Comment:
This is the content of media section description. So if you want to put the media icon click event link, there is no need to add the link here.